sig-scheduling/charter.md

```diff
@@ -0,0 +1,67 @@
+# SIG Scheduling Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG Scheduling is responsible for the components that make Pod placement decisions.
+We build Kubernetes schedulers and scheduling features for Pods. We design and
+implement features that allows users to customize placement of Pods on the nodes
+of a cluster. These features include those that improve reliability of workloads,
+more efficient use of cluster resources, and/or enforces placement policies. 
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Scheduling related features (e.g. Node Affinity)
+- Kube-scheduler performance and scalability (with [sig-scalability](../sig-scalability))
+- Kube-scheduler reliability (problem detection and remediation)
+- Pod scheduling APIs (with [sig-api-machinery](../sig-api-machinery))
+- Node-level resource management (with [sig-node](../sig-node))
+- Cluster-level resource management (with [sig-node](../sig-node) and [sig-storage](../sig-storage))
+- Pod scheduling policies (with [wg-policy](../wg-policy))
+
+**This is NOT** a list of specific code locations,
+  or projects. For those refer to [SIG Subprojects][sig-subprojects].
+
+#### Cross-cutting and Externally Facing Processes
+
+- Kube-scheduler [test grid] and [perf dashboard]
+
+### Out of scope
+
+- network management ([sig-network](../sig-network))
+- persistent storage management ([sig-storage](../sig-storage))
+- enforcement of resource quota and other admission policies
```

Comment:
are you assuming this is sig-apimachinery?

if so, we need to make sure that is covered in their charter.  /cc @smarterclayton @bgrant0607 who are primary/secondary reviewers for that once its posted.

Reply:
I am not sure if SIG scheduling was responsible for developing and maintaining the resource quota feature in the past. It seems to be more aligned with resource management workgroup, but it is fine with me to have it under SIG scheduling.

Reply:
AFAIK, "resource quota feature" are **NOT** in scheduling SIG but I do not know who's owner. Anyway, I'm also ok to make it under scheduling umbrella.

Reply:
personally, i think it makes more sense for quota enforcement to be under api-machinery charter, the enforcement works like the garbage collector.

/cc @deads2k @lavalamp -

Reply:
I definitely consider api machinery to own at least the object count quota. CPU & RAM are maybe not us unless they are done generically. Feel free to add the relevant OWNERs to one of api machinery's subprojects if you like.

Reply:
Thanks, Daniel. I update the doc accordingly.